### PR TITLE
Forward anew5

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: test/unit/module/derivative/CMakeLists.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Copyright : Fluxion Project Contributors
 ## SPDX-License-Identifier: BSL-1.0
 ##==================================================================================================
-add_subdirectory(bessel)
+##add_subdirectory(bessel)
```
Comment (jfalcou) [inline test/unit/module/derivative/CMakeLists.txt:6]: Why ?

File: test/unit/module/forward_automatic/core/absmax.cpp [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  TTS_EQUAL(der(eve::absmax(a0, vda1))      , derivative_2nd(eve::absmax)(a0, a1));
+
+  TTS_EQUAL(val(eve::absmax(vda0, a1, a2))  , eve::absmax(a0, a1, a2));
+  TTS_EQUAL(val(eve::absmax(a0, vda1, a2))  , eve::absmax(a0, a1, a2));
+  TTS_EQUAL(val(eve::absmax(a0, a1, vda2))  , eve::absmax(a0, a1, a2));
+  TTS_EQUAL(der(eve::absmax(vda0, a1, a2))  , derivative_1st(eve::absmax)(a0, a1, a2));
+  TTS_EQUAL(der(eve::absmax(a0, vda1, a2))  , derivative_2nd(eve::absmax)(a0, a1, a2));
+  TTS_EQUAL(der(eve::absmax(a0, a1, vda2))  , derivative_3rd(eve::absmax)(a0, a1, a2));
+
+//    TTS_EQUAL(val(eve::absmax[mask](vda0, a1, a2))  , eve::absmax[mask](a0, a1, a2));
```
Comment (jfalcou) [inline test/unit/module/forward_automatic/core/absmax.cpp]: Why ?

File: test/unit/module/derivative/CMakeLists.txt (hunk 1 above)
Comment (jtlap) [reply test/unit/module/derivative/CMakeLists.txt:6]: Not finished ulps are not good

File: test/unit/module/forward_automatic/core/absmax.cpp (hunk 2 above)
Comment (jtlap) [reply test/unit/module/forward_automatic/core/absmax.cpp]: decommented

**Pull request merged**